res/analytics.js

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+if(
+  // Only include on perf-html.io
+  window.location.host.includes('perf-html.io') &&
+  // Observe do not track.
+  (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack) === '1'
```

Comment:
don't you want the opposite ? ;)
It's '1' if the user actually specified "I don't want to be tracked" ;)

That's why my suggested code was:
```javascript
const doNotTrack = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack) === '1';
if (!doNotTrack) { ... }
```
(note the `!` :) )